crates/transaction-emitter/src/lib.rs

```diff
@@ -70,6 +71,7 @@ impl Default for EmitThreadParams {
             wait_millis: 0,
             wait_committed: true,
             txn_expiration_time_secs: 30,
+            do_not_check_stats_at_end: false,
```

Comment:
Having this negative flag is a bit confusing to understand. How about we change this to `check_stats_at_end` and set the default to `true` instead?